src/Test.Utilities/Test.Utilities.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp3.1;net472</TargetFrameworks>
     <NonShipping>true</NonShipping>
+    <IsShipping>false</IsShipping>
```

Comment:
Where are we using `NonShipping`? Can you please file a separate issue to tracking consolidating `NonShipping` and `IsShipping` to a single property?

Reply:
#6388